lib/reverse_words.rb

```diff
@@ -1,6 +1,31 @@
 # A method to reverse each word in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2), where n is the length of my_words
```

Comment:
Incorrect. The outer loop is O(n), and all the individual calls to string_reverse together add up to another O(n), so it's O(n) overall.